### DISCUSSION TITLE
Support for NXP S32K3xx platform

### DISCUSSION THREAD

**Discussion by Sathishkumar-rk** [zephyrproject-rtos/zephyr#55295] [General]
Hi @danieldegrasse @DerekSnell,

Is there any plan to support S32K3xx platform in zephyr project?

Regards,
Sathishkumar K

**Comment by DerekSnell** [comment]
Hi @Sathishkumar-rk ,
At the moment, NXP does not have plans to upstream Zephyr support for the S32K3xx family.  It is being discussed internally at NXP, and this may come later.  But no plans at the moment.

Best regards

**Reply by Sathishkumar-rk** [reply]
Thanks for the information @DerekSnell, hope the support will get added in future releases.

**Reply by igalloway** [reply]
@Sathishkumar-rk   I would be interested in knowing more details about your interest in S32K3 and Zephyr. If you don't mind sharing, would you mind messaging me directly? iain.galloway@nxp.com

**Comment by DerekSnell** [comment]
Hi @Sathishkumar-rk ,
FYI, there is a new PR [Introduce support for NXP MR-CANHUBK3 board](https://github.com/zephyrproject-rtos/zephyr/pull/58332/files), which adds new device support for the S32K344.  Best regards

**Reply by Sathishkumar-rk** [reply]
Thanks for the update @DerekSnell

**Comment by igalloway** [comment]
FYI the MR-CANHUB344 board is in inventory now. Official launch in a couple of days.
We'll also include extra documentation linked from NXP.gitbook.io/hovergames

This is just the first PR for zephyr on K3, there's a schedule for additional peripheral support over the next few months.

Happy to answer any questions I can.

***@***.***

On May 30, 2023 9:40 AM, Derek Snell ***@***.***> wrote:
Caution: This is an external email. Please take care when clicking links or opening attachments. When in doubt, report the message using the 'Report this email' button


Hi @Sathishkumar-rk<https://github.com/Sathishkumar-rk> ,
FYI, there is a new PR Introduce support for NXP MR-CANHUBK3 board<https://github.com/zephyrproject-rtos/zephyr/pull/58332/files>, which adds new device support for the S32K344. Best regards

**Reply by Sathishkumar-rk** [reply]
Thanks for the update @igalloway.

Happy to hear that zephyr supporting S32Kxx platform. 

> I might come up with bank of questions regarding this platform support, I'm glad you are already mentioned about it   :v:

Thanks again
